modules/tinymce/src/themes/silver/main/ts/ui/menus/menubar/Integration.ts

```diff
@@ -37,17 +37,17 @@ const make = (menu: { title: string; items: string[] }, registry: MenuRegistry,
     getItems: () => Arr.bind(menu.items, (i): Menu.NestedMenuItemContents[] => {
       const itemName = i.toLowerCase();
       if (itemName.trim().length === 0) {
-        return [ ];
+        return [];
```

Comment:
Any particular reason these where changed?

Reply:
I think the auto formatter did that